run-build-functions.sh

```diff
@@ -59,6 +60,10 @@ mkdir -p $NETLIFY_CACHE_DIR/.gimme_cache/gocache
 mkdir -p $NETLIFY_CACHE_DIR/.homebrew-cache
 mkdir -p $NETLIFY_CACHE_DIR/.cargo
 
+
+# tmp dir
+NETLIFY_TMP_DIR=$(mktemp -d)
```

Comment:
Needed a `tmp` dir to be used for the `.workspace_locations` file and figured I should probably add a generic dir to be used for other purposes if needed. Let me know what you all think or if there's any concerns with this. CC @vbrown608

Reply:
No longer applicable, see - https://github.com/netlify/build-image/pull/526#discussion_r581135288